site/blog/starlingx-meetup-september-2019.md

```diff
@@ -0,0 +1,48 @@
+
+After the opening further presenters shared their tech solutions or product practices with StarlingX on the infrastructure layer.
+
+### Presentations
+
+**“StarlingX Practice in Industrial Manufacturing Edge Cloud”:**
+Besides being a traditional telecom operator, China Unicom is also pursuing CSP business opportunities on certain market segments, like industrial manufacturing on the edge. These environments demand platforms to connect the industrial fields with the cloud data centers with using edge and IOT technologies. The presentation outlined how IaaS, PaaS and SaaS solutions can provide a layered architecture to fulfill the needs of this use case.
+
+StarlingX is one of the favorable solutions on the IaaS layer, benefited from its dedicated optimizations for edge, such as, low latency, real-time, security and high availability, in addition to the orchestration capabilities inherit from Kubernetes and OpenStack. The speaker also highlighted an example how the Distributed Cloud feature can be used to deploy a cross-regional platform to fulfill the needs of the industrial manufacturing use case.
+
+**"Multi-cloud and edge platform convergence":**
+The second presenter from FiberHome was sharing information about their product history and plans and activities with StarlingX. Over the years the company has been creating a full portfolio for cloud (FitCloud) and edge (FitEC) use cases for different cloud models including public, private and hybrid solutions. To support the technology evolution and remain innovative in the field FiberHome has decided to move to the direction of converging multi-cloud and edge platform solutions to improve resource utilization and efficiency. For the edge piece they have evaluated several projects available in the open soure space and selected StarlingX for the infrastructure layer. Since 2018 they have been actively contributing to the community and also have a member in the StarlingX Technical Steering Committee (TSC).
+
+**"Edge platform for smart agriculture":**
+This case study was quite interesting and attracted a big portion of the audience, not only because of its vivid example (of cultivating shrimps), but more about the realistic difficulties of landing the edge platform in fields (around shrimp ponds). The presenter from 99Cloud were explaining how they took advantage of StarlingX capabilities to address these challenges. Several features were highlighted during the talk, like 'All-in-One Simplex' which is a deployment scenario that StarlingX supports which lets you install the platform only on one server. StarlingX can easily scale from that deployment to a multi-server platform depending on the needs of the use case. The Distributed Cloud feature was also highlighted that they used to manage dozens of hosts with great experience.
```

Comment:
The presenters from 99Cloud were explaining how they took advantage of StarlingX capabilities to address these challenges. Several features were highlighted during the talk, like 'All-in-One Simplex' which is a deployment scenario that StarlingX supports which lets you install the platform only on one server. StarlingX can easily scale from that deployment to a multi-server platform depending on the needs of the use case. The Distributed Cloud feature was also highlighted that they used to manage dozens of hosts with great experience.